### PR TITLE
Update billing and subscription FAQ

### PR DESCRIPTION
**Pull request by stephaurelio**
### Proposed changes

This PR adds several questions to the billing and subscription FAQ pages that were sourced from support KB articles. 

Deploy Previews:
- [Billing FAQ](https://deploy-preview-17757--docsdocker.netlify.app/billing/faqs/)
- [Subscription FAQ](https://deploy-preview-17757--docsdocker.netlify.app/subscription/faq/)

**Notes for reviewers**

- Support reviewer: check for accuracy, if there are any insights from past tickets that should be considered with this content
- Technical reviewer: verify technical accuracy, that the content is correct
- Docs reviewer: check against the style guide, grammar, etc.

### Related issues (optional)

**Jira ticket:** ENGDOCS-1513

@netlify /faq/

### CODE REVIEW

File: billing/faqs.md [hunk 1]
```diff
@@ -59,4 +59,34 @@ A coupon can be used when you:
 
 You are asked to enter your coupon code when you confirm or enter your payment method. 
 
-If you use a coupon to pay for a subscription, when the coupon expires, your payment method is charged the full cost of your subscription. If you do not have a saved payment method, your account is downgraded to a free subscription. 
+If you use a coupon to pay for a subscription, when the coupon expires, your payment method is charged the full cost of your subscription. If you do not have a saved payment method, your account is downgraded to a free subscription.
+
+### How do I include my VAT number on my invoice?
+
+Organization owners can update invoice details to include a VAT number. To update the billing information for your organization:
```
Comment (ob1dev) [inline billing/faqs.md:66]: **suggestion:** VAT number can be added to both user and organization account as long as a billing address is in EU zone (see `eu_vat` here https://stripe.com/docs/billing/customer/tax-ids).

```suggestion
User account or organization owners can update invoice details to include a VAT number. To update the billing information for your organization:
```

**Review by ob1dev — APPROVED** [review]
LGTM 👍🏻

File: billing/faqs.md [hunk 2]
```diff
@@ -59,4 +59,34 @@ A coupon can be used when you:
...
+3. In the **Billing Information** section, enter your information in the **VAT/Tax ID** field.
+4. Select **Update**.
+
+To download your updated invoice:
+1. Go to **Billing History**, select the **Actions** icon, and then select **Update invoice**.
+2. Select **Update and download** to download the invoice that includes the VAT number.
+
+### How do I update the email that receives billing invoices?
+
+You can update the billing information for your user account or owners can update this information for organizations.
```
Comment (aevesdocker) [inline billing/faqs.md:79]: Super picky point but I don't think you need the introductory sentence. I think you can dive straight in. I feel the question header already sets the context enough

File: billing/faqs.md [hunk 3]
```diff
@@ -59,4 +59,34 @@ A coupon can be used when you:
 
 You are asked to enter your coupon code when you confirm or enter your payment method. 
 
-If you use a coupon to pay for a subscription, when the coupon expires, your payment method is charged the full cost of your subscription. If you do not have a saved payment method, your account is downgraded to a free subscription. 
+If you use a coupon to pay for a subscription, when the coupon expires, your payment method is charged the full cost of your subscription. If you do not have a saved payment method, your account is downgraded to a free subscription.
+
+### How do I include my VAT number on my invoice?
+
```
Comment (aevesdocker) [inline billing/faqs.md:65]: Same comment as below:

> Super picky point but I don't think you need the introductory sentence. I think you can dive straight in. I feel the question header already sets the context enough

**Review by aevesdocker — APPROVED** [review]
Nice! Just a small non-blocking comment from me

**Review by d-droyalty — APPROVED** [review]
No issues with the information that I can find

File: billing/faqs.md (hunk 2 above)
Comment (stephaurelio) [reply billing/faqs.md:79]: That makes sense! Want to keep this as brief as possible. Will update!

File: billing/faqs.md (hunk 3 above)
Comment (stephaurelio) [reply billing/faqs.md:65]: Updating!

File: billing/faqs.md (hunk 1 above)
Comment (stephaurelio) [reply billing/faqs.md:66]: Good to know, will update.

**Review by aevesdocker — APPROVED** [review]
:shipit:

**Pull request merged**